### DISCUSSION TITLE
Add chunk splitting

### DISCUSSION THREAD

**Discussion by mahyarkarimi** [meteor/meteor#12039] [Feature Requests]
Is it possible to have the meteor app built into chunks instead of a large bundle? it makes the performance of the app much better and faster to load, with less network traffic.

**Comment by afrokick** [comment]
Did you try dynamic imports? https://docs.meteor.com/packages/dynamic-import.html

**Reply by Saeeed-B** [reply]
@afrokick @mahyarkarimi 
This is not really enough, documents and examples should be more abundant and practical.

**Reply by boomfly** [reply]
> Did you try dynamic imports? https://docs.meteor.com/packages/dynamic-import.html

This not working in web.cordova build.

In my project i have app.js 16Mb release size.

And app load time ~15-20 sec

**Comment by boomfly** [comment]
Any new news about dynamic-import for web.cordova target?
